src/js/pages/Profile/index.jsx

```diff
@@ -36,6 +38,12 @@ class Profile extends Component {
   render() {
     const {profile, prices, address} = this.props;
     if(!profile || !prices) return <Loading page={true} />;
+
+    const toBN = web3.utils.toBN;
+    const relayGasPrice = toBN(this.props.gasPrice || '0').mul(toBN(120)).div(toBN(100)); // 120%. toBN doesnt like decimals?
+    const onlyETH = toBN(web3.utils.toWei(this.props.ethBalance || '0', 'ether')).lt(toBN(500000).mul(relayGasPrice)); // only allow ETH if less than 500000*gasPrice
+    const filteredOffers = onlyETH ? profile.offers.filter(x => x.token.symbol === 'ETH') : profile.offers;
```

Comment:
Can we put these filters and functions to a shared util?

Reply:
good idea!